metadatafields.md

```diff
@@ -45,4 +45,45 @@ Example: <https://dspace7.4science.it/dspace-spring-rest/#https://dspace7.4scien
 
 The supported parameters are:
 * **(mandatory)** schema, the prefix of the metadata schema (i.e. "dc", "dcterms", "eperson, etc.)
-* page, size [see pagination](README.md#Pagination)
\ No newline at end of file
+* page, size [see pagination](README.md#Pagination)
+
+## Creating a Metadata Field
+
+**POST /api/core/metadatafields?schemaId=<:schemaId>**
```

Comment:
I prefer to take an uniform decision about how to reference the "parent" object during the creation, see the corresponding comment in https://github.com/DSpace/Rest7Contract/pull/33#discussion_r240558102

I guess that the way suggested here (to use a query parameter) is the easier/quick to implement

Reply:
We will use the approach from 
https://github.com/DSpace/Rest7Contract/pull/33#discussion_r
This corresponds to what has been implemented here, so this PR already uses that approach